hw2/04-background.js

```diff
@@ -1 +1,40 @@
 // Add your code here
```

Comment:
The assignment sheet wants you to start the page with the background already changing every 3 seconds. You will need a page load event listener.